public/codenotes.php

```diff
@@ -178,9 +178,7 @@ function saveCodeNote(rowIndex, isDeleting = false) {
             rowEl.querySelector('td[data-current-author]').dataset.currentAuthor = currentUsername;
 
             const authorAvatarSpan = authorAvatarCellEl.querySelector('span.inline.whitespace-nowrap');
-            const newXInitValue = `attachTooltipToElement($el, { dynamicType: 'user', dynamicId: '${currentUsername}', dynamicContext: '' })`;
-            authorAvatarSpan.setAttribute('x-init', newXInitValue);
-            attachTooltipToElement(authorAvatarSpan, { dynamicType: 'user', dynamicId: currentUsername, dynamicContext: '' });
+            authorAvatarSpan.removeAttribute('@mouseover');
```

Comment:
It is too much of a hassle to keep juggling the user prop on the tooltip as we're dynamically swapping it. Since the prop is just being changed to ourselves when we edit a code note, remove showing the tooltip altogether.